src/python/pants/core/util_rules/environments.py

```diff
@@ -379,7 +374,7 @@ class OptionField(field_type, EnvironmentSensitiveOptionFieldMixin):  # type: ig
         # TODO: use envvar-like normalization logic here
         alias = f"{scope}_{option.flag_names[0][2:]}".replace("-", "_")
         required = False
-        value: field_value_type  # type: ignore[valid-type]
+        value: Any
```

Comment:
Good change. It doesn't really matter because rules should not be directly consuming the fields via the Target API. It's all abstracted away via the Subsystem helper code